apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/trade_wizard/review/TradeWizardReviewView.java

```diff
@@ -124,7 +125,7 @@ class TradeWizardReviewView extends View<StackPane, TradeWizardReviewModel, Trad
         priceDescription.getStyleClass().add(descriptionStyle);
         gridPane.add(priceDescription, 0, rowIndex);
 
-        price = new MultiStyleLabelPane();
+        price = new TextFlow();
```

Comment:
Actually I don't see a reason why TextFlow is needed here. We have a rather static layout without expected linebreaks.